### DISCUSSION TITLE
dbx limitation multitask job still actual?

### DISCUSSION THREAD

**Discussion by markonian-117** [databrickslabs/dbx#333] [Q&A] [answered]
Fellow Experts,

Can you please help confirm the following limitation "dbx execute cannot be used with multi-task jobs" which is mentioned in the Databricks Documentation https://docs.databricks.com/dev-tools/dbx.html#limitations but not on the GitHub page of the DBX project.

 Is this limitation still applicable or has it been resolved? Thank you

Regards,
Mark

**Comment by renardeinside** [comment]
hi @mvervuurt , 
yes, this limitation in still in-place. We're working on the next release which will cover multitask jobs for `dbx execute`.

**Comment by renardeinside** [comment, marked as answer]
Upd: multitask jobs are fully supported in 0.7.0 🔥
